dashboard/app/views/shared/_school_info.html.haml

```diff
@@ -89,9 +89,9 @@
 
   $(document).ready(function() {
     var options = #{existing_school_info.to_json};
-    options['suppressScrolling'] = #{suppress_scrolling}
+    options['suppressScrolling'] = #{defined? suppress_scrolling && suppress_scrolling}
 
-    if (#{assume_usa}) {
```

Comment:
This `defined? && var` pattern can be shortened into one command with [Object.presence](http://apidock.com/rails/Object/presence).